code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -484,12 +472,14 @@ func (sc *StorageSmartContract) selectBlobbers(
 }
 
 type updateAllocationRequest struct {
-	ID           string           `json:"id"`              // allocation id
-	OwnerID      string           `json:"owner_id"`        // Owner of the allocation
-	Size         int64            `json:"size"`            // difference
-	Expiration   common.Timestamp `json:"expiration_date"` // difference
-	SetImmutable bool             `json:"set_immutable"`
-	UpdateTerms  bool             `json:"update_terms"`
+	ID               string           `json:"id"`              // allocation id
+	OwnerID          string           `json:"owner_id"`        // Owner of the allocation
+	Size             int64            `json:"size"`            // difference
+	Expiration       common.Timestamp `json:"expiration_date"` // difference
+	SetImmutable     bool             `json:"set_immutable"`
+	UpdateTerms      bool             `json:"update_terms"`
+	AddedBlobberId   string           `json:"added_blobber_id"`
+	RemovedBlobberId string           `json:"removed_blobber_id"`
```

Comment:
Why do we use the past tense of `Added`, `Removed`, this makes me think when they are added or removed, and what will this request do to them. If this is a request to add or remove blobber for an allocation, perhaps we can just use AddBlobberID or RemoveBlobberID.